uber/templates/groupform.html

```diff
@@ -103,14 +103,7 @@
         <p class="help-block col-sm-9 col-sm-offset-3">The one you want us to link on our website, or n/a</p>
     </div>
 
-    {% if not c.COLLECT_FULL_ADDRESS %}
-    <div class="form-group">
-        <label for="address" class="col-sm-3 control-label">Address and Phone Number</label>
-        <div class="col-sm-6">
-            <textarea class="form-control" name="address" rows="4" placeholder="Your or your business' full address and phone number for tax purposes.">{{ group.address }}</textarea>
-        </div>
-    </div>
-    {% endif %}
+    {{ macros.address_form(group=group, maybe_collision=c.PAGE_PATH in ['/preregistration/dealer_registration'] or attendee_or_group.badge_type == c.PSEUDO_DEALER_BADGE) }}
```

Comment:
I debated whether to make this set of address fields a macro or a template to be included. This is the strongest argument for the latter -- `groupform.html` doesn't always have access to `is_prereg_dealer` (defined in regform.html) so I had to basically redefine 'on the dealer page' in this function call. It's messier than I'd like. :/